.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,14 @@
+name: Release notes
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I suppose we don't want to add v4-dev here on purpose?

Reply:
Think so. Makes me think we'd want to setup a separate workflow just for that branch?